Not able to save long error message on failed workspace start

### Description
```
017-12-28 11:56:59,893[aceSharedPool-2]  [ERROR] [o.e.c.a.w.s.WorkspaceRuntimes 292]   - Error response from docker API, status: 500, message: Failed to Setup IP tables: Unable to enable SKIP DNAT rule:  (iptables failed: iptables --wait -t nat -I DOCKER -i br-7793c1060f47 -j RETURN: iptables: No chain/target/match by that name.
 (exit status 1))

org.eclipse.che.api.workspace.server.spi.InternalInfrastructureException: Error response from docker API, status: 500, message: Failed to Setup IP tables: Unable to enable SKIP DNAT rule:  (iptables failed: iptables --wait -t nat -I DOCKER -i br-7793c1060f47 -j RETURN: iptables: No chain/target/match by that name.
 (exit status 1))

        at org.eclipse.che.workspace.infrastructure.docker.network.NetworkLifecycle.createNetwork(NetworkLifecycle.java:46)
        at org.eclipse.che.workspace.infrastructure.docker.DockerInternalRuntime.internalStart(DockerInternalRuntime.java:164)
        at org.eclipse.che.api.workspace.server.spi.InternalRuntime.start(InternalRuntime.java:109)
        at org.eclipse.che.api.workspace.server.WorkspaceRuntimes$StartRuntimeTask.run(WorkspaceRuntimes.java:266)
        at org.eclipse.che.commons.lang.concurrent.CopyThreadLocalRunnable.run(CopyThreadLocalRunnable.java:29)
        at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1626)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.eclipse.che.infrastructure.docker.client.exception.DockerException: Error response from docker API, status: 500, message: Failed to Setup IP tables: Unable to enable SKIP DNAT rule:  (iptables failed: iptables --wait -t nat -I DOCKER -i br-7793c1060f47 -j RETURN: iptables: No chain/target/match by that name.
 (exit status 1))

        at org.eclipse.che.infrastructure.docker.client.DockerConnector.getDockerException(DockerConnector.java:1518)
        at org.eclipse.che.infrastructure.docker.client.DockerConnector.createNetwork(DockerConnector.java:1289)
        at org.eclipse.che.workspace.infrastructure.docker.network.NetworkLifecycle.createNetwork(NetworkLifecycle.java:39)
        ... 8 common frames omitted
2017-12-28 11:56:59,917[aceSharedPool-2]  [WARN ] [o.e.c.a.w.s.WorkspaceManager 425]    - Cannot set error status of the workspace workspacevr7h2fffparwy2hx. Error is: Exception [EclipseLink-4002] (Eclipse Persistence Services - 2.7.0.v20170811-d680af5): org.eclipse.persistence.exceptions.DatabaseException
Internal Exception: org.h2.jdbc.JdbcSQLException: Value too long for column "ATTRIBUTES VARCHAR(255)": "STRINGDECODE('Error response from docker API, status: 500, message: Failed to Setup IP tables: Unable to enable SKIP DNAT rule: ... (260)"; SQL statement:
INSERT INTO workspace_attributes (workspace_id, attributes, attributes_key) VALUES (?, ?, ?) [22001-196]
Error Code: 22001
Call: INSERT INTO workspace_attributes (workspace_id, attributes, attributes_key) VALUES (?, ?, ?)
        bind => [3 parameters bound]
Query: DataModifyQuery(sql="INSERT INTO workspace_attributes (workspace_id, attributes, attributes_key) VALUES (?, ?, ?)")
^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B^[[B2017-12-28 11:59:03,079[nio-8080-exec-4]  [INFO ] [o.e.c.a.w.s.WorkspaceRuntimes 229]   - Starting workspace 'che/wksp-node' with id 'workspacevr7h2fffparwy2hx' by user 'che'
2017-12-28 11:59:25,979[aceSharedPool-3]  [INFO ] [o.e.c.a.w.s.WorkspaceRuntimes 269]   - Workspace 'che:wksp-node' with id 'workspacevr7h2fffparwy2hx' started by user 'che'
```

--- a/wsmaster/che-core-sql-schema/src/main/resources/che-schema/6.0.0/11__increase_workspace_attributes_values_length.sql
+++ b/wsmaster/che-core-sql-schema/src/main/resources/che-schema/6.0.0/11__increase_workspace_attributes_values_length.sql
@@ -1,0 +1,12 @@
+--
+-- Copyright (c) 2012-2017 Red Hat, Inc.
+-- All rights reserved. This program and the accompanying materials
+-- are made available under the terms of the Eclipse Public License v1.0
+-- which accompanies this distribution, and is available at
+-- http://www.eclipse.org/legal/epl-v10.html
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+ALTER TABLE workspace_attributes ALTER COLUMN attributes TYPE TEXT;

